about/process/ppa.rst

```diff
@@ -1,33 +0,0 @@
-UBports Debian repository
-=========================
-
-repo.ubports.com
-----------------
-
-This is the legacy package repository for UBports projects.  It hosts the Debian packages of the current stable release.
-
-repo2.ubports.com
```

Comment:
just to double check: we don't need to document repo2 anymore?!

the docs used to say repo is legacy and repo2 is current. PR says there's just repo

Reply:
repo.ubports.com has been updated to point to the same place as Aptly-based repo2.ubports.com by Marius since some time ago. In other words, the original freight-based repo.ubports.com is no longer available.

(This is one instance of out-of-date information before the recent changes I mentioned in the Telegram chat)